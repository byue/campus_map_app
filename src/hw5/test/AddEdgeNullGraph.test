# Tests that adding an edge to a null graph results in expected output.

# Add edge to null graph
AddEdge graph1 n1 n2 a