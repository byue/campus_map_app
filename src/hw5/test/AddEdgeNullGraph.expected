# Tests that adding an edge to a null graph results in expected output.

# Add edge to null graph
graph1 has not been created